lib/api_catalogue.rb

```diff
@@ -10,6 +11,56 @@ def self.from_csv(catalogue_csv:, organisation_csv:)
     new(apis: apis, organisations: organisations)
   end
 
+  def self.from_url(url)
```

Comment:
I think we should split this out into a separate class, so `from_url` mirrors [the way that the `FetcherService` works](https://github.com/co-cddo/federated-api-model/blob/main/examples/java/spring-boot/src/main/java/uk/gov/api/springboot/domain/services/fetcher/FetcherService.java) and calls out to a `V1AlphaFetcher` that implements `v1alpha`, etc